supervisor/ipsetsupervisor_test.go

```
@@ -29,33 +31,34 @@ func TestNewIPSetSupervisor(t *testing.T) {
 	pe := mockenforcerDefaultFQConfig(t)
 	ipt := provider.NewTestIptablesProvider()
 	ips := provider.NewTestIpsetProvider()
+	ipu, _ := iptablesutils.NewIpsetUtils(ipt, ips)
```

Same as above, don't ignore the error